src/client/pages/Landing/landingPageData.ts

```diff
@@ -25,11 +27,13 @@ type ProductData = {
 
 type ProductsData = Record<Market, ProductData>
 
-export const productsData: ProductsData = {
+export const getProductsData = (): ProductsData => ({
```

Comment:
Convert to a function since `checkFeature` is accessing window but the module is loaded on the server (for some reason...)